apps/web/src/app/api/checker/regions/_checker.ts

```diff
@@ -0,0 +1,77 @@
+import { Receiver } from "@upstash/qstash/cloudflare";
```

Comment:
What's the benefit over using `@upstash/qstash/nextjs`?

Reply:
We are in the edge environment which is the same as cloudfare worker 

We cant use all the node api